.gitignore

```diff
@@ -21,3 +21,4 @@ _savebackup/
 *.swatches
 /imgui.ini
 
+
```

Comment:
Nitpick: It looks like a whitespace change snuck in. This should be removed.

Reply:
Hi @AMZN-puvvadar, yes I'll remove it for sure :) Should I make a new commit (along with other changes you mentioned) with a whitespace removed?

Reply:
@AMZN-puvvadar I have removed this whitespace in the latest commit. Do let me know if you need any other changes 👍